test/uk/gov/hmrc/merchandiseinbaggage/connectors/MibConnectorSpec.scala

```diff
@@ -17,35 +17,35 @@
 package uk.gov.hmrc.merchandiseinbaggage.connectors
 
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.merchandiseinbaggage.config.MibConfiguration
 import uk.gov.hmrc.merchandiseinbaggage.model.api.GoodsDestinations.GreatBritain
 import uk.gov.hmrc.merchandiseinbaggage.model.api._
 import uk.gov.hmrc.merchandiseinbaggage.model.api.calculation._
-import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub._
+import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub
 import uk.gov.hmrc.merchandiseinbaggage.utils.DataModelEnriched._
 import uk.gov.hmrc.merchandiseinbaggage.{BaseSpecWithApplication, CoreTestData}
 
 import java.time.LocalDateTime
 import java.time.temporal.ChronoUnit
 
-class MibConnectorSpec extends BaseSpecWithApplication with CoreTestData with MibConfiguration {
+class MibConnectorSpec extends BaseSpecWithApplication with CoreTestData {
 
   private val client             = app.injector.instanceOf[MibConnector]
+  private val stub               = app.injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these stub changes, thanks